packages/get-starknet/src/utils/helper.ts

```diff
@@ -0,0 +1,30 @@
+import type { Call } from 'starknet';
+import type { Call as CallGetStarknetV4 } from 'starknet-types-07';
+
+/**
+ * Converts an array of calls from either the `CallGetStarknetV4[]` format
+ * or the standard `Call[]` format into the standard `Call[]` format. If the input
+ * calls are already in the correct format, no changes are made.
+ *
+ * The function ensures that:
+ * - `contract_address` from `CallGetStarknetV4` is renamed to `contractAddress` if needed.
+ * - `entry_point` from `CallGetStarknetV4` is renamed to `entrypoint` if needed.
+ * - `calldata` is set to an empty array if undefined.
+ *
+ * @template T - The type of the calls array, either `CallGetStarknetV4[]` or `Call[]`.
+ * @param calls - The array of call objects to be formatted.
+ * @returns The array of formatted calls in the `Call[]` format.
+ */
+export const formatCalls = <CallType extends Call | CallGetStarknetV4>(calls: CallType[]): Call[] => {
```

Comment:
it does not looks like u need a generic type, as u only support 2 types, which is quite specific

```suggestion
export const formatCalls = (calls: Call[] | CallGetStarknetV4[]): Call[] => {
```